client/src/components/DisplayTable/DisplayTable.tsx

```diff
@@ -421,6 +429,43 @@ export class _DisplayTable extends React.Component<
       />
     );
 
+    const cols_with_row_grouping = [
+      "prgm",
+      "so",
+      "desc",
+      "region",
+      "tp",
+      "fol",
+    ];
+    const trigger_column = _.intersection(
+      cols_with_row_grouping,
+      _.keys(column_configs)
+    )[0];
+
+    const row_grouped_data = _.chain(data)
+      .groupBy("dept")
+      .map((dept_data) => {
+        const basic_columns = ["dept", "prgm", "legal_title", trigger_column];
+        const summable_columns = _.chain(dept_data[0])
+          .omit(basic_columns)
+          .keys()
+          .value();
+        const column_sums = summable_columns.map((col) => {
+          return _.sumBy(dept_data, col);
+        });
+        const columns = _.zipObject(summable_columns, column_sums);
```

Comment:
Above two could be written as
```
_.chain(summable_columns)
          .map((col) => [col, _.sumBy(dept_data, col)])
          .fromPairs()
          .value();
```
Or you could even put `summable_columns` into the above chain